src/_base/helm/app/templates/service/php-fpm-exporter/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+{{ if .Values.service.php_fpm_exporter }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.resourcePrefix }}php-fpm-exporter
+  labels:
+    app.service: {{ .Values.resourcePrefix }}php-fpm-exporter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.service: {{ .Values.resourcePrefix }}php-fpm-exporter
+  template:
+    metadata:
+      labels:
+        app.service: {{ .Values.resourcePrefix }}php-fpm-exporter
+    spec:
+      containers:
+      - image: hipages/php-fpm_exporter
```

Comment:
Should this not be in the same pod as nginx/php-fpm ("web" deployment)
Otherwise php-fpm would need to be exposed to the network.

Reply:
Good point.